src/cascadia/TerminalApp/TerminalTab.h

```diff
@@ -83,6 +83,9 @@ namespace winrt::TerminalApp::implementation
         winrt::Windows::UI::Xaml::Controls::MenuFlyoutItem _closeTabsAfterMenuItem{};
         winrt::TerminalApp::TabHeaderControl _headerControl{};
 
+        Windows::Foundation::Collections::IVector<uint16_t> _mruPanes;

```

Comment:
Does this need to be a whole WinRT vector? Is an `std::vector` sufficient for this since it won't be sent around in any winRT APIs?

Reply:
(a winrt vector is somewhat more expensive than a normal vector, since it's a whole heap-allocated object that can be reference-counted and stuff)

Reply:
(and all functions that operate on it do so through virtual function dispatch)